python/cudf/cudf/_lib/reduce.pyx

```diff
@@ -123,3 +125,28 @@ def scan(col_inp, col_out, scan_op, inclusive):
     free_column(c_col_out)
 
     return
+
+
+def group_std(key_columns, value_columns, ddof=1):
```

Comment:
Why is this in reduce rather than groupby?

Reply:
The idea is that, similar to quantiles, functionalities stay in `src/<functionality>` and groupby gets to call an internal API. It was done in quantiles because it shared code with the series implementation and I can imagine others would share too with their series-level counterparts. This one in particular doesn't but keeping with the program, it was put here.